specification/eventgrid/data-plane/readme.md

```diff
@@ -147,56 +165,6 @@ input-file:
 - Microsoft.ApiCenter/stable/2018-01-01/ApiCenter.json
 ```
 
-### Tag: package-2023-11-01
-
-These settings apply only when `--tag=package-2023-11-01` is specified on the command line.
-
-``` yaml $(tag) == 'package-2023-11-01'
-input-file:
-- Microsoft.Storage/stable/2018-01-01/Storage.json
-- Microsoft.EventHub/stable/2018-01-01/EventHub.json
-- Microsoft.Resources/stable/2018-01-01/Resources.json
-- Microsoft.EventGrid/stable/2023-11-01/EventGrid.json
-- Microsoft.EventGrid/stable/2018-01-01/SystemEvents.json
-- Microsoft.DataBox/stable/2018-01-01/DataBox.json
-- Microsoft.Devices/stable/2018-01-01/IotHub.json
-- Microsoft.ContainerRegistry/stable/2018-01-01/ContainerRegistry.json
-- Microsoft.ServiceBus/stable/2018-01-01/ServiceBus.json
-- Microsoft.Media/stable/2018-01-01/MediaServices.json
-- Microsoft.Maps/stable/2018-01-01/Maps.json
-- Microsoft.AppConfiguration/stable/2018-01-01/AppConfiguration.json
-- Microsoft.SignalRService/stable/2018-01-01/SignalRService.json
-- Microsoft.KeyVault/stable/2018-01-01/KeyVault.json
-- Microsoft.MachineLearningServices/stable/2018-01-01/MachineLearningServices.json
-- Microsoft.Cache/stable/2018-01-01/RedisCache.json
-- Microsoft.Web/stable/2018-01-01/Web.json
-- Microsoft.Communication/stable/2018-01-01/AzureCommunicationServices.json
-- Microsoft.PolicyInsights/stable/2018-01-01/PolicyInsights.json
-- Microsoft.ContainerService/stable/2018-01-01/ContainerService.json
-- Microsoft.ApiManagement/stable/2018-01-01/APIManagement.json
-- Microsoft.HealthcareApis/stable/2018-01-01/HealthcareApis.json
-- Microsoft.AVS/stable/2018-01-01/PrivateCloud.json
-
-```
-
-### Tag: package-2024-01-01
```

Comment:
I think these have to stay because swagger avocado check, can make a diff readme